internal/rbd/rbd_journal.go

```diff
@@ -598,16 +604,23 @@ func RegenerateJournal(
 	if err != nil {
 		return "", err
 	}
+
 	if imageData != nil {
 		rbdVol.ReservedID = imageData.ImageUUID
 		rbdVol.ImageID = imageData.ImageAttributes.ImageID
 		rbdVol.Owner = imageData.ImageAttributes.Owner
+		rbdVol.RbdImageName = imageData.ImageAttributes.ImageName
```

Comment:
I dont think we need this line

Reply:
SetMetadata() needs it https://github.com/ceph/ceph-csi/blob/devel/internal/rbd/rbd_util.go#L1663